runtime/rastrace/mt.tdf

```diff
@@ -23,7 +23,7 @@
 Executable=mt
 DATFileName=J9TraceFormat.dat
 
-TraceEntry=Trc_MethodEntry   Overhead=1 Level=5 Group=bytecodeMethods                Template="%.*s.%.*s%.*s bytecode method, this = 0x%zx"
+TraceEntry=Trc_MethodEntry   Overhead=1 Level=5 Group=bytecodeMethods                Template="%.*s.%.*s%.*s bytecode method, this = %.*s 0x%zx,"
```

Comment:
You need to make a new trace entry since you are change the format. You can add the `obsolete` tag to this one.

Reply:
I think we want to tread carefully here: I won't be surprised if `mt.1-12` are well known tracepoint ids.